src/components/articles.jsx

```diff
@@ -26,7 +27,9 @@ export const Articles = () => {
         {articles.map((article) => {
           return (
             <div key={article.article_id}>
-              <h2>{article.title}</h2>
+              <Link to={`articles/${article.article_id}`}>
```

Comment:
THIS HERE - Need to be <Link to={`/articles/${article.article_id}`}>

The path needs to start with a '/' otherwise it won't work.